sdk/keyvault/keyvault-admin/src/accessControlModels.ts

```diff
@@ -114,7 +149,13 @@ export interface KeyVaultRoleAssignmentProperties {
  * A scope of the role assignment.
  * The valid scopes are: "/", "/keys" and any a specific resource Id followed by a slash, as in "ID/".
  */
-export type RoleAssignmentScope = "/" | "/keys" | string;
+export type KeyVaultRoleAssignmentScope = "/" | "/keys" | string;
+
+/**
+ * A scope of the role definition.
+ * The valid scopes are: "/" or "/keys"
+ */
+export type KeyVaultRoleScope = "/" | "/keys";
```

Comment:
@sadasant let me know how rude this is! It's a breaking change, so I can revert it, but I thought it might be ok since we haven't GA'd this? I'd love to stay consistent here if possible!

Reply:
Nothing new in betas is breaking till GA. Until it ships for the first time, break away! 😉